src/njoy21/input/ACER/Card9/Nbint.hpp

```diff
@@ -0,0 +1,15 @@
+struct Nbint {
+  using Value_t = int;
+  static std::string name(){ return "nbint"; }
+
+  static std::string description() {
+    return
+        "The nbint argument specifies thenuber of bins for incoherent\n"
```

Comment:
What, you don't want this to look like it was designed by a non-English speaker?